Bio/SeqIO/SffIO.py

```diff
@@ -1306,4 +1311,5 @@ def write_record(self, record):
 
 if __name__ == "__main__":
     from Bio._utils import run_doctest
-    run_doctest(verbose=0)
\ No newline at end of file
+
```

Comment:
Extra blank line?

(But putting in the trailing newline at the end of the file is good)